.overcommit.yml

```diff
@@ -26,7 +26,7 @@ PreCommit:
     requires_files: true
   StandardJS:
     enabled: true
-    command: ['standard']
+    command: ['yarn', 'run', 'standard']
```

Comment:
I think @naseberry just updated this so might wanna check with him if this change is all good